akka-docs-dev/rst/java/migration-guide-1.0-2.x-java.rst

```diff
@@ -422,7 +422,7 @@ should be replaced by:
 .. includecode:: code/docs/MigrationsJava.java#query-param
 
 SynchronousFileSource and SynchronousFileSink
-============================================
+=============================================
```

Comment:
Well, now it will output proper HTML ;) (yes, RST actually does care of the length of those bars)